### PR TITLE
[DOCS] added virtual box guide

### PR DESCRIPTION
**Pull request by MaheshBharadwaj**
Adds guide to run skiftOS on virtual box

### CODE REVIEW

**Comment by sleepy-monax** [comment]
Nice guide I like the GIFs :+1:

**Comment by MaheshBharadwaj** [comment]
> Nice guide I like the GIFs 👍

Thanks :D anything else to add?

File: readme.md [hunk 1]
```diff
@@ -5,7 +5,8 @@
   <a href="https://discord.gg/gamGsfg">Discord</a> -
   <a href="https://skiftos.org/">Website</a> -
   <a href="https://github.com/skiftOS/skift/releases">Releases</a> -
-  <a href="manual/readme.md">Manual</a>
+  <a href="manual/readme.md">Manual</a> -
+  <a href="manual/running_vm.md">Run as VM</a>
```
Comment (sleepy-monax) [inline readme.md:9]: There is already a link to this file in the readme, no need for two ;)

File: manual/running_vm.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+## Creating the VM on Virtual Box
+
+5. Create a new VM by clicking on the **New** button.
+6. Give the OS a meaningful name and set **Type: Other** and **Version: Other/Unknown** and click on next.
+7. Allocate a minimum of **128MB RAM** _(512MB recommended)_ and click on next.
+8. Choose **Do not add a virtual hard disk**, click on _Create_ and ignore the warning message and press **Continue**.
+
+  ![Steps 5-8 gif](./screenshots/vm_step1.gif)
+
+## Final few Steps!
```
Comment (sleepy-monax) [inline manual/running_vm.md:21]: I don't think this header is really needed

File: readme.md (hunk 1 above)
Comment (MaheshBharadwaj) [reply readme.md:9]: to which, manual or Run as VM both are below the header and here too

File: manual/running_vm.md (hunk 2 above)
Comment (MaheshBharadwaj) [reply manual/running_vm.md:21]: okay but the gif can remain in the same place?

**Comment by sleepy-monax** [comment]
I added some comments on this PR ;)

File: manual/running_vm.md (hunk 2 above)
Comment (sleepy-monax) [reply manual/running_vm.md:21]: Yes ;)

File: readme.md (hunk 1 above)
Comment (sleepy-monax) [reply readme.md:9]: The manual link can remain but run as vm need to go

**Comment by sleepy-monax** [comment]
LGTM

**Pull request merged**